### ISSUE TITLE
JAWS does not recognize the toggle switch

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#182]
It seems that JAWS does not support aria-label (at least for checkboxes with role=switch).  Here is the html:

``` html
<input id="402" role="switch" type="checkbox" aria-checked="false" aria-label="‪Two Balloon Experiment‬" style="width: 1px;">
```

JAWS does not recognize this.   All other AT recognizes as a switch or at least a check box.

**Comment by jessegreenberg** [comment]
From the Paciello group (https://www.paciellogroup.com/blog/2015/10/notes-on-use-of-multiple-aria-role-attribute-values/)

> A new role switch has been introduced in ARIA 1.1, for assistive technology that recognise the switch role it should be the role used to represent the semantics of the element, for those that don’t it should be ignored and the checkbox role should be recognised as representing the semantics of the element.

**Comment by jessegreenberg** [comment]
Ah...further down in the article:

> JAWS does not recognise the switch role and does not fallback to reporting as a checkbox. This is a JAWS bug.

@terracoda how would you like to proceed?  The article and our tests show that the switch is not supported in JAWS.  Should we revert to a check box or a button with the aria-pressed attribute?

**Comment by terracoda** [comment]
@jessegreenberg, hmm, this is drag! The switch seems to be working quite nicely with other screen readers. If it is not read out at all by JAWS users, than it is a serious problem, and it would be wise to revert. @jessegreenberg, would it be possible to set up a test page that has all three options. I don't remember which one sounds best.
Need to check notes because changing the control means also changing the label. I think the checkbox works best for current label.

**Comment by jessegreenberg** [comment]
@terracoda we could set up a test page, but I think the fastest way to do this would be to create test examples in a regular HTML page and decide what sounds best there before placing in the sim.  I can do this if you want.  The three options are checkbox, button with aria-pressed, and what else?

**Comment by jessegreenberg** [comment]
Marking as high priority, this should be done before #188.

**Comment by jessegreenberg** [comment]
Interviews will not be using JAWS, so this is no longer a high priority issue.

**Comment by terracoda** [comment]
@jessegreenberg, you are right. This is not a high priority. A test page will be useful for more scenarios as well. We can set this up after the interviews. The cases are:
1. Toggle Button 
[ARIA Toggle button](http://files.paciellogroup.com/blogmisc/ARIA/togglebutton.html)
[Toggle Buttons on MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role)
2. I think checkbox is best used with role=switch. Without switch, we use the toggle button (above).
3. Simple button with a changing label (Add green balloon / Remove green balloon). This would be more like the Remove/Add Wall button - a simple button is operated by both enter key and space bar. I don't think we ever tried this one with actual users.
4. A list of two Radio buttons. This is a  very verbose option and really not ideal in this situation.

**Comment by jessegreenberg** [comment]
Since JAWS does not recognize role=switch, we are using aria-pressed instead.
